telegram-villa.py

```diff
@@ -4,20 +4,24 @@
 import os
 
 TOKEN = os.getenv("TOKEN")
+APITOKEN = 'JQVl8GNrhQzVhdf7hLVnq1pw6BqvxOGUNhfyBsMm0Ty9H5chvGjYD85m1vhi'
+
+URL = 'https://api.sportsmonks.com/api/v2.0'
+TEAM = 'Aston Villa'
```

Comment:
worth having a scan of this https://realpython.com/python-constants/. These caps variables are constants (sort of). you've used them pretty well here!